[2025-10-30T11:44:38.022+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-30T11:43:00+00:00 [queued]>
[2025-10-30T11:44:38.065+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-30T11:43:00+00:00 [queued]>
[2025-10-30T11:44:38.067+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-30T11:44:38.114+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-30 11:43:00+00:00
[2025-10-30T11:44:38.135+0000] {standard_task_runner.py:60} INFO - Started process 3848 to run task
[2025-10-30T11:44:38.153+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-30T11:43:00+00:00', '--job-id', '1659', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmp908lgq3x']
[2025-10-30T11:44:38.154+0000] {standard_task_runner.py:88} INFO - Job 1659: Subtask ConsumKafka
[2025-10-30T11:44:38.296+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-30T11:43:00+00:00 [running]> on host 46f6cfd4d166
[2025-10-30T11:44:38.593+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-30T11:43:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-30T11:43:00+00:00'
[2025-10-30T11:44:38.607+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-30T11:44:38.625+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.8', 9092)]>: Broker version identified as 2.6
[2025-10-30T11:44:38.629+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-30T11:44:38.633+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-30T11:44:38.636+0000] {dag1_kafka_processing.py:66} INFO - ‚úÖ Consommateur Kafka cr√©√© pour le topic 'source'. En attente de messages...
[2025-10-30T11:44:38.643+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-30T11:44:38.646+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-30T11:44:38.647+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-30T11:44:38.755+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-30T11:44:38.758+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-30T11:44:38.760+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-30T11:44:38.762+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-30T11:44:38.764+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-30T11:44:38.770+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.8', 9092)]>: connecting to kafka:9092 [('172.18.0.8', 9092) IPv4]
[2025-10-30T11:44:38.771+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Connection complete.
[2025-10-30T11:44:38.874+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-30T11:44:38.880+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-e5fd9b35-6546-469b-99bd-c1d00e2a0abb for group ***-consumers-dag1; will retry join-group
[2025-10-30T11:44:38.882+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-30T11:44:38.888+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-30T11:44:40.955+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 467 (member_id: kafka-python-2.2.15-e5fd9b35-6546-469b-99bd-c1d00e2a0abb, protocol: range)>
[2025-10-30T11:44:40.983+0000] {subscription_state.py:254} INFO - Updated partition assignment: []
[2025-10-30T11:44:40.985+0000] {consumer.py:249} INFO - Setting newly assigned partitions () for group ***-consumers-dag1
[2025-10-30T11:44:43.640+0000] {dag1_kafka_processing.py:81} INFO - ‚è≥ Aucun message encore... nouvelle tentative.
[2025-10-30T11:44:45.645+0000] {base.py:951} WARNING - Heartbeat failed for group ***-consumers-dag1 because it is rebalancing
[2025-10-30T11:44:45.648+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-30T11:44:45.649+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-30T11:44:45.659+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 468 (member_id: kafka-python-2.2.15-e5fd9b35-6546-469b-99bd-c1d00e2a0abb, protocol: range)>
[2025-10-30T11:44:45.660+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-30T11:44:45.666+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-30T11:44:45.667+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-30T11:44:45.679+0000] {dag1_kafka_processing.py:77} INFO - üì© Message re√ßu : {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.378715146429919, 'latitude': 48.83978812460128, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.6972763033474223, 'latitude': 40.38569037069795, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-30T11:44:45.686+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-30T11:44:45.688+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-30T11:44:45.699+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-30T11:44:45.700+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-30T11:44:45.701+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.8', 9092)]>: Closing connection. 
[2025-10-30T11:44:45.704+0000] {python.py:201} INFO - Done. Returned value was: {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.378715146429919, 'latitude': 48.83978812460128, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.6972763033474223, 'latitude': 40.38569037069795, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-30T11:44:45.771+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251030T114300, start_date=20251030T114438, end_date=20251030T114445
[2025-10-30T11:44:45.853+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-30T11:44:45.936+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
